src/registrar/assets/sass/_theme/_containers.scss

```diff
@@ -0,0 +1,6 @@
+@use "uswds-core" as *;
+
+.grid-container--widescreen,
+.usa-identifier__container--widescreen {
+    max-width: var(--widescreen-max-width) !important;
```

Comment:
You should not need !important if you match the USWDS selectors exactly, and it'll be safer.